src/test/run-make/unicode-input/Makefile

```diff
@@ -1,26 +0,0 @@
--include ../tools.mk
-
-# This test attempts to run rustc itself from the compiled binary; but
-# that means that you need to set the LD_LIBRARY_PATH for rustc itself
-# while running multiple_files, and that won't work for stage1.
-
-# FIXME ignore windows
-ifndef IS_WINDOWS
-ifeq ($(RUST_BUILD_STAGE),1)
-DOTEST=
-else
-DOTEST=dotest
-endif
-endif
-
-all: $(DOTEST)
-
-dotest:
-	# check that we don't ICE on unicode input, issue #11178
```

Comment:
Actually, I'm just going to file "write more unicode tests" and put it on myself.  We need more than 1 or 2 anyways, which is all we really had before.